libraries/AP_SmartAudio/AP_SmartAudio.cpp

```diff
@@ -0,0 +1,947 @@
+#include "AP_SmartAudio.h"
+#include <AP_Math/crc.h>
+#include <GCS_MAVLink/GCS.h>
+
+#define AP_VTX_DEV_MODE     // DEV TESTS ACTIVATE
+#ifdef AP_VTX_DEV_MODE
+# define debug(fmt, args...)	hal.console->printf("\r " fmt "\n", ##args)
+//#define debug(fmt,args...) gcs().send_text(MAV_SEVERITY_DEBUG , fmt, ##args);
+#else
+//# define debug(fmt, args...)	do {} while(0)
+#endif
+
+extern const AP_HAL::HAL &hal;
+const char *TAG="VTX-SMARTAUDIO ";
+
+
+// table of user settable parameters
+const AP_Param::GroupInfo AP_SmartAudio::var_info[] = {
+
+    // @Param: ENABLED
+    // @DisplayName: Enable SmartAudio protocol on VTX
+    // @Description: Enable SmartAudio protocol on VTX
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Advanced
+    AP_GROUPINFO_FLAGS("ENABLED",     1, AP_SmartAudio::configured_smartaudio_params, enabled, 0,AP_PARAM_FLAG_ENABLE),
```

Comment:
So this should be "ENABLE" not "ENABLED" but I don't think you need it - you should activate this feature based on the presence of "VTX_ENABLE" and a serial port being configured with smart audio "35". Otherwise the user has to set 3 flags to get this going which is too much

Reply:
Fixed i remove it